webapp/src/components/jira_field.jsx

```diff
@@ -29,6 +29,10 @@ export default class JiraField extends React.PureComponent {
                     style={getStyle().jiraIcon}
                     src={allowedValue.iconUrl}
                 />
+                {/* this works because allowedValue test cases have */}
```

Comment:
nit: Can we put the comments above the fragment? I dislike this comment style. It looks like you're passing comments into the component. :)

Reply:
0/5.  We probably have a MM doc style guide and will check this after pushing through with these changes!